content/rc/security/cidr-whitelist.md

```diff
@@ -14,11 +14,15 @@ The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) [allow
 
 You can configure your database's CIDR allow list to restrict client connections to a specific range of IP addresses.
 
+## Prerequisites
```

Comment:
Instead of having this be its own section, I would recommend using a note like the one in [VPC peering](https://docs.redis.com/staging/DOC-2819/rc/security/vpc-peering/#:~:text=Note%3A%20VPC%20peering%20is%20available%20only%20with%20Flexible%20or%20Annual%20subscriptions.%20It%20is%20not%20supported%20for%20Fixed%20or%20Free%20subscriptions.)